src/rust/ide/view/graph-editor/src/component/visualization/foreign/java_script/instance.rs

```diff
@@ -182,6 +192,13 @@ impl InstanceModel {
         self.root_node.set_size(size);
     }
 
+    fn on_hide(&self) {
+        if let Some(f) = &*self.on_hide {
+            let context = &JsValue::NULL;
+            let _ = f.call0(context);
```

Comment:
Why we skip the result? If this is Result, I think we should log a warning if it is an Error.

Reply:
I just did this to be consistent with the other JS calls in the module